flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-arches": [
+        "aarch64"
```

Comment:
Fedora used to package id3lib also for aarch64. What was the error message?

Reply:
It's [here](https://flathub.org/builds/#/builders/45/builds/2378/steps/6/logs/stdio) if you want to read through the whole thing, but the part that sticks out to me is `configure: error: cannot guess build type; you must specify one`.

Reply:
People either do something like this https://github.com/flathub/org.xiphos.Xiphos/blob/53a76df702435cad1b901bdf4e3b654771f17169/org.xiphos.Xiphos.json#L109 or remove these files and run autoreconf to regenerate them.

Reply:
I tried the first approach in commit de2f878, which resulted in the same error. I'm not sure how to proceed with the second approach, so I tried just replacing the config.guess file with a fresh one from GNU in commit e0cf045, and now the build succeeds.

Since that file is hosted by GNU in a git repository I can change this so it downloads directly from there, right now the file is just directly in this repository.